Readme.md

```diff
@@ -121,7 +131,7 @@ root_token = s.Cmro41vNI4wIndgrPqzlqOKY
 Then deploy to the cluster:
 
 ```shell
-./liquid deploy
+./liquid.py deploy
```

Comment:
Do we really want to drop the `./liquid $something` scheme and add `.py` everywhere? We'll have to update all our instructions. At least let's keep it around for a while as "deprecated" and have it print a warning. cc @gabriel-v

Reply:
So, what should I do? @gabriel-v @mgax 
- make `./liquid` a wrapper
- consider `./liquid` deprecated and display a warning
- leave it as it currently is

Reply:
Make `./liquid` a wrapper for `pipenv run python liquid.py`. Which means it won't auto-install packages the first time, like it used to, but we can document that initial `pipenv install` step.